service/templates/search.html

```diff
@@ -19,12 +19,21 @@ <h1 class="heading-xlarge form-title">Find a title</h1>
                           <input type="text" class="form-control" name="search_term" id="search_term">
                       </div>
                       <div class="form-group">
-                          <input type="submit" value="Search" class="button">
+                          <input id="search_button" type="submit" value="Search" class="button">
                       </div>
                   </form>
                 </div>
             </div>
-
         </div>
     </main>
+    <script type="text/javascript">
+        function ga_title_search() {
+            search_term = document.getElementById('search_term').value
+            title_number_regex = '^([A-Z]{0,3}[1-9][0-9]{0,5}|[0-9]{1,6}[ZT])$'
+            if (search_term.match(title_number_regex)) {
+              ga('send', 'event', 'Search', 'TitleNumber', search_term);
+            }
+        }
+        document.getElementById('search_button').onclick = ga_title_search;
```

Comment:
a) would like to chat about how we are planning on including Python variables in JavaScript then? see the other script in particular that use page numbers etc... should we just be extracting those from HTML elements instead? (think that would be better actually)

b) I disagree about document loading since we only want to track the search term once it's been typed in and the search_button clicked on